### PR TITLE
Hide portfolio content when wallet is not connected

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v3/src/i18n/translations/it.json [hunk 1]
```diff
@@ -115,12 +115,18 @@
     "titleNoPositions": "Niente in Portafogli",
     "titleProjectedYield": "",
     "titleTotalPortfolioValue": "",
-    "titleTotalYieldEarned": "Rendimento Totale"
+    "titleTotalYieldEarned": "Rendimento Totale",
+    "titleWalletNotConnected": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/it.json:119]: `Wallet non collegato`

File: packages/tempus-client_v3/src/i18n/translations/it.json [hunk 2]
```diff
@@ -115,12 +115,18 @@
...
+    "descriptionWalletNotConnected": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/it.json:120]: `Connetti il tuo wallet per gestire il portafoglio.`

File: packages/tempus-client_v3/src/i18n/translations/it.json [hunk 3]
```diff
@@ -115,12 +115,18 @@
...
+    "buttonConnectWallet": ""
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/it.json:121]: `Connetti Wallet`

File: packages/tempus-client_v3/src/i18n/translations/es.json [hunk 4]
```diff
@@ -115,12 +115,18 @@
...
+    "buttonConnectWallet": ""
   },
   "PortfolioPositions": {
     "buttonBrowseMarkets": "Ir a Mercados",
     "descriptionNoPositions": "Tu billetera está vacía. Hacer algo.",
-    "titleNoPositions": "Sin posición"
+    "titleNoPositions": "Sin posición",
+    "titleWalletNotConnected": "",
+    "descriptionWalletNotConnected": "",
+    "buttonConnectWallet": ""
```
[the remark this answers is not in the document]
Comment (lazarmitic) [reply packages/tempus-client_v3/src/i18n/translations/es.json]: Updated

File: packages/tempus-client_v3/src/i18n/translations/it.json (hunk 3 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/i18n/translations/it.json]: Updated

File: packages/tempus-client_v3/src/i18n/translations/it.json (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/i18n/translations/it.json]: Updated

File: packages/tempus-client_v3/src/i18n/translations/it.json (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/i18n/translations/it.json]: Updated

File: packages/tempus-client_v3/src/i18n/translations/es.json [hunk 5]
```diff
@@ -115,12 +115,18 @@
...
   },
   "PortfolioPositions": {
     "buttonBrowseMarkets": "Ir a Mercados",
     "descriptionNoPositions": "Tu billetera está vacía. Hacer algo.",
-    "titleNoPositions": "Sin posición"
+    "titleNoPositions": "Sin posición",
+    "titleWalletNotConnected": "",
+    "descriptionWalletNotConnected": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/es.json:128]: `Wallet no conectado`

File: packages/tempus-client_v3/src/i18n/translations/es.json [hunk 6]
```diff
@@ -115,12 +115,18 @@
...
+    "buttonConnectWallet": ""
   },
   "PortfolioPositions": {
     "buttonBrowseMarkets": "Ir a Mercados",
     "descriptionNoPositions": "Tu billetera está vacía. Hacer algo.",
-    "titleNoPositions": "Sin posición"
+    "titleNoPositions": "Sin posición",
+    "titleWalletNotConnected": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/es.json:127]: `Wallet no conectado`

File: packages/tempus-client_v3/src/i18n/translations/es.json [hunk 7]
```diff
@@ -115,12 +115,18 @@
...
+    "descriptionWalletNotConnected": "",
+    "buttonConnectWallet": "Conectar Wallet"
   },
   "PortfolioPositions": {
     "buttonBrowseMarkets": "Ir a Mercados",
     "descriptionNoPositions": "Tu billetera está vacía. Hacer algo.",
-    "titleNoPositions": "Sin posición"
+    "titleNoPositions": "Sin posición",
+    "titleWalletNotConnected": "",
+    "descriptionWalletNotConnected": "",
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/i18n/translations/es.json:128]: `Conecte su wallet para administrar la billetera.`

File: packages/tempus-client_v3/src/i18n/translations/es.json (hunk 6 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/i18n/translations/es.json]: Updated

File: packages/tempus-client_v3/src/i18n/translations/es.json (hunk 7 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/i18n/translations/es.json]: Updated

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**